test/Microsoft.ML.Tests/SelectColumnsTransformsTests.cs

```diff
@@ -0,0 +1,302 @@
+﻿using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Tools;
+using Microsoft.ML.TestFramework;
+using Microsoft.ML.Transforms;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests
+{
+    public class SelectColumnsTransformsTests : BaseTestClass
```

Comment:
>BaseTestClass [](start = 48, length = 13)

Please add a test that calls `TestEstimatorCore`

Reply:
Or add that to one of the existing tests.

---
In reply to: [225386549](https://github.com/dotnet/machinelearning/pull/1269#discussion_r225386549) [](ancestors = 225386549)